libdnf5/module/module_sack_impl.hpp

```diff
@@ -114,7 +114,7 @@ class ModuleSack::Impl {
     /// @param module_items Module Items to resolve.
     /// @return `std::pair` of problems in resolving and ModuleErrorType.
```

Comment:
May I ask you for an update of a description of the return value?

Reply:
Thanks, updated